include/zpc/aes_xts.h

```diff
@@ -54,6 +54,26 @@ int zpc_aes_xts_set_key(struct zpc_aes_xts *ctx, struct zpc_aes_key *key1,
  */
 __attribute__((visibility("default")))
 int zpc_aes_xts_set_iv(struct zpc_aes_xts *ctx, const unsigned char *iv);
+/**
+ * Get the intermediate initialization vector to be used in the context
+ * of an AES-XTS operation.
+ * \param[in,out] ctx AES-XTS context
+ * \param[out] iv application provided buffer with 16 bytes size to
+ * receive the 16 byte intermediate iv 
+ * \return 0 on success. Otherwise, a non-zero error code is returned.
+ */
+__attribute__((visibility("default")))
+int zpc_aes_xts_get_intermediate_iv(struct zpc_aes_xts *ctx, unsigned char iv[16]);
```

Comment:
I still find it a bit inconsistent that the already existing zpc_aes_xts_set_iv() has `const unsigned char *iv`, but the new zpc_aes_xts_get/set_intermediate_iv() have `unsigned char iv[16]` although both take 16 bytes....

While the array type better describes what is expected, it is still different to the existing functions ....
Of course, we can't change the existing ones to use an array without breaking the API.

Should we maybe stick with the `unsigned char *iv` version also for the new functions?
Or am I just to picky? ;-)

Reply:
> Should we maybe stick with the unsigned char *iv version also for the new functions?

Hm, yes, this is what I initially intended. I'll check again ...